docs/configuration-reference/components/velero.md

```diff
@@ -86,6 +86,8 @@ component "velero" {
   # restic {
   #   credentials = file("cloud-credentials-file")
   #
+  #   default_volumes_to_restic = true
```

Comment:
I know this is upstream name for this flag, but perhaps we could come up with some better name for it. For example, we're already in `restic` block, so there is no need to add `to_restic` IMO.

Maybe we could have something like `require_volume_annotation`? And we set it to true by default?

Reply:
I agree on the naming, but I am on the fenc with setting the value as `true` by default as its not the default behavior of Velero.

Reply:
No no, `require_volume_annotation = true` would result in `default_volumes_to_restic = false`.